test/unit/cinstance_test.rb

```diff
@@ -488,11 +488,15 @@ class SuspendTest < ActiveSupport::TestCase
   end
 
   test 'with_trial_period_expired_or_accepted' do
-    assert_equal 0, Cinstance.with_trial_period_expired_or_accepted(Date.today).count
```

Comment:
Right! This randomly fails because there should be an application created with master_account.
And sometimes master_account is left in DB. Only jenkins issue though